.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+# Pull Request Description
+- Please provide a short description of the feature.
+
+## Why Is This Feature/Fix Necessary?
+- Please provide a short description of why it is necessary.
+
+## PR Type
+What kind of change does this PR introduce?
+
+- [] fix
+- [] Feature
+- [] Code style update
+- [] Refactoring
+- [] CI related changes
+- [] Documentation content changes
+- [] Tests
+- [] Other
```

Comment:
**nitpick** 🦦
Usually the content of the PR should be one or more of the ones shown them and is already present in the branch name, so I consider this should not be here. Maybe could be more useful to change it to **Jira Ticket**

<sub><sup>I'm using [conventional comments](https://conventionalcomments.org/) formatting to improve comments readability and reader's expectations in this PR.</sup></sub>